core/src/test/java/io/grpc/internal/AutoConfiguredLoadBalancerFactoryTest.java

```diff
@@ -99,7 +99,7 @@ public class AutoConfiguredLoadBalancerFactoryTest {
               new FakeLoadBalancerProvider("test_lb2", testLbBalancer2, nextParsedConfigOrError2)));
 
   private final Class<? extends LoadBalancer> pfLbClass =
-      PickFirstLoadBalancerProvider.enableNewPickFirst
+      PickFirstLoadBalancerProvider.isEnabledNewPickFirst()
```

Comment:
What's with this change? I don't see the point in it unless you also make `enableNewPickFirst` private.

Reply:
Added the step of making enableNewPickFirst private